include/cuco/sentinel.cuh

```diff
@@ -17,7 +17,7 @@
 #pragma once
 
 namespace cuco {
-namespace sentinel {
+inline namespace sentinel {
```

Comment:
This could be a PR against `dev` branch.

Reply:
/done #247